src/common/profileSection/profileSection.module.css

```diff
@@ -3,13 +3,11 @@
   --title-fontsize: var(--fontsize-h-3);
   background: var(--color-white);
   border-radius: 2px;
-  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
   padding: var(--spacing-l) var(--horizontal-whitespace);
   margin-bottom: var(--spacing-xl);
 }
 
 .profileSectionTitleRow {
-  align-items: baseline;
```

Comment:
I assume that this was by intention, but I'm commenting to highlight just to be sure.

Reply:
Yeah it was. `profileSectionTitleRow` had align-items set twice.